### PR TITLE
Fix NSIDC download issues

### PR DESCRIPTION
**Pull request by alex-s-gardner**

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -143,7 +143,6 @@ function netrc!(username, password)
     open(fn, "a") do f
         write(f, "\n")
         write(f, "machine urs.earthdata.nasa.gov login $username password $(password)\n")
-        write(f, "machine n5eil01u.ecs.nsidc.org login $username password $(password)\n")
```
Comment (evetion) [inline src/utils.jl:146]: I think this is the only change needed in this PR.

**Review by evetion — CHANGES_REQUESTED** [review]

**Comment by alex-s-gardner** [comment]
Sorry for all the commits, I've now discovered `]dev` so my pull request etiquette should improve

**Comment by evetion** [comment]
No problem, but you're doing everything in the same branch. As in this PR/branch you have:

- Fix for ICESat-2 download (original issue, PR title)
- EarthDataCloud search (and adding the search bbox to those granules)
- Bounding box filtering on ATL06

Ideally those are separate PRs, that makes it easier to discuss a single issue and merge when ready. Now we can't merge the download fix, because the other parts are unfinished.

**Comment by alex-s-gardner** [comment]
I'll clean this up before requesting a pull

**Pull request closed without merging**